_docs/style/spacing.md

```diff
@@ -85,86 +105,70 @@ Exceptions are made for some elements such as dashboard panels and [cards]({{sit
 
 {% column left:"hxCol hxSpan-12" %}
 
-<!-- BUG: a[href="content"] apparently is a specifically styled thing in HelixUI... -->
+## Content spacing
 
-## Content text
+Content is any information displayed inside a container. When you vertically stack content components, ensure that you conform to line height and bottom margin spacing values. Bottom margins contain extra spacing that improves visually perceptible intervals, which is helpful for conveying hierarchy.
 
-Content lives inside the container. Content is stacked vertically using line-heights and bottom margins (depending on type of content). Bottom margins add extra spacing that improves visually perceptible intervals, which is helpful for conveying hierarchy. Content contains:
+There are two types of content:
 
-- Headers are at the highest level of hierarchy for content (page title, section title, sub-section title, container title)
-- Just below the header hierarchy is the data interspersed in the form of paragraphs, lists, forms, and tables that we will refer to as leaf-nodes.
+- **Headers**: Headers are the highest level of the content hierarchy and are used for page titles, section titles, subsection titles, and  container titles.
+- **Body elements**: Body elements are located just below the header and consist of list items, drop-down selectors, and paragraphs.
```

Comment:
these aren't the ONLY things. this is just an example list...
could include visualizations, cards, form controls, and body text.

maybe we should generalize to example categories. 
 Body elements are located below the header and consist of a wide variety of components including body text, list items, form controls, and visualizations.

Reply:
LEt's bundle this into a follow up story

Reply:
Done.